aspnetcore/security/blazor/index.md

```diff
@@ -0,0 +1,478 @@
+
+The Blazor server-side project template can set up authentication for you when the project is created.
+
+# [Visual Studio](#tab/visual-studio)
+
+Follow the Visual Studio guidance in the <xref:blazor/get-started> article to create a new Blazor server-side project with an authentication mechanism.
+
+After choosing the **Blazor (server-side)** template in the **Create a new ASP.NET Core Web Application** dialog, select **Change** under **Authentication**.
+
+A dialog opens to offer the same set of authentication mechanisms available for other ASP.NET Core projects:
+
+* **No Authentication**
+* **Individual User Accounts** &ndash; User accounts can be stored:
+  * Within the app using ASP.NET Core's [Identity](xref:security/authentication/identity) system.
+  * With [Azure AD B2C](xref:security/authentication/azure-ad-b2c).
+* **Work or School Accounts**
+* **Windows Authentication**
+
+# [Visual Studio Code](#tab/visual-studio-code)
+
+Follow the Visual Studio Code guidance in the <xref:blazor/get-started> article to create a new Blazor server-side project with an authentication mechanism:
+
+```console
+dotnet new blazorserverside -o {APP NAME} -au {AUTHENTICATION}
+```
+
+Permissible authentication values (`{AUTHENTICATION}`) are shown in the following table.
+
+| Authentication mechanism                                                                 | `{AUTHENTICATION}` value |
+| ---------------------------------------------------------------------------------------- | :----------------------: |
+| No Authentication                                                                        | `None`                   |
+| Individual<br>Users stored in the app with ASP.NET Core Identity.                        | `Individual`             |
+| Individual<br>Users stored in [Azure AD B2C](xref:security/authentication/azure-ad-b2c). | `IndividualB2C`          |
+| Work or School Accounts<br>Organizational authentication for a single tenant.            | `SingleOrg`              |
+| Work or School Accounts<br>Organizational authentication for multiple tenants.           | `MultiOrg`               |
+| Windows Authentication                                                                   | `Windows`                |
+
+The command creates a folder named with the value in the `{APP NAME}` placeholder and uses the same value as the app name. For more information, see the [dotnet new](/dotnet/core/tools/dotnet-new) command in the .NET Core Guide.
+
+Open the project's folder in Visual Studio Code.
+
+<!--
+
+# [Visual Studio for Mac](#tab/visual-studio-mac)
+
+1. Follow the Visual Studio for Mac guidance in the <xref:blazor/get-started> article.
+
+1.
+
+1.
+
+-->
+
+# [.NET Core CLI](#tab/netcore-cli/)
```

Comment:
Use VSC or CLI, not both. Probably go with VSC. The pure command line folks can figure out what to do with VSC instructions.

cc @tdykstra  @scottaddie

Reply:
It think we've combining tabs. We get a content validation error ... a false positive ... but it works. I'll do it on the next commit.

Reply:
You don't have to mention CLI as a tab, just have the VS and VSC tabs.

Reply:
I admit that I should've followed the guidance that I was given earlier (a combined VSC + CLI tab) ... it's hard to get everything exactly so on the first pass with a new topic 🏃😅, but that was what I was told to do earlier. I have done that on a few topics now. I take it that you (the team) is changing that guidance?

Reply:
Yes, from experience with combined tabs we've concluded that making a combined VSC/CLI tab is more trouble than it's worth.

Reply:
I see. Thanks for letting me know.

@danroth27 You said that the VSC experience is going to change. IIRC when we set up the *Get Started* topic, I split the tabs out over the same concern. I anticipated different guidance at some preview release along the way to RTM.

Reply:
Let's do this right now: I'll comment out the tab but not delete it. If we hear back shortly from Dan that we shouldn't have a CLI tab *ever*, then I'll delete it.

I wish they would be left even when the content is mostly duplicated. If I'm a CLI person, I want *my tab that applies to my scenario* right there ... no guesswork about what I'm supposed to do. Anyway ... I'll at least temporarily drop it, then I'll kill it totally if that's the best way to go.